lib/beaker-hostgenerator/data/vmpooler.rb

```diff
@@ -35,6 +35,10 @@ module Vmpooler
           'platform' => 'el-7-x86_64',
           'template' => 'centos-7-x86_64'
         },
+        'cumulus2-64' => {
```

Comment:
@joshcooper I went ahead and changed it to `cumulus25-64` since we don't know how versioning works for the Cumulus platforms and may need as you suggest to support 2.6 in the future.